src/lib/ngsiNotify/Notifier.h

```diff
@@ -45,34 +62,32 @@ class Notifier
 public:
   virtual ~Notifier(void);
 
-  virtual void sendNotifyContextRequest(NotifyContextRequest&            ncr,
+  virtual void sendNotifyContextRequest(ContextElementResponse*          notifyCerP,
                                         const ngsiv2::Notification&      notification,
-                                        const std::string&               tenant,
+                                        const notifStaticFields&         nsf,
                                         long long                        maxFailsLimit,
                                         long long                        failsCounter,
-                                        const std::string&               xauthToken,
-                                        const std::string&               fiwareCorrelator,
-                                        unsigned int                     correlatorCounter,
                                         RenderFormat                     renderFormat,
                                         const std::vector<std::string>&  attrsFilter,
                                         bool                             blacklist,
                                         bool                             covered,
                                         const std::vector<std::string>&  metadataFilter);
 
 protected:
-  static std::vector<SenderThreadParams*>* buildSenderParams(NotifyContextRequest&            ncr,
-                                                             const ngsiv2::Notification&      notification,
-                                                             const std::string&               tenant,
-                                                             long long                        maxFailsLimit,
-                                                             long long                        failsCounter,
-                                                             const std::string&               xauthToken,
-                                                             const std::string&               fiwareCorrelator,
-                                                             unsigned int                     correlatorCounter,
-                                                             RenderFormat                     renderFormat,
-                                                             const std::vector<std::string>&  attrsFilter,
-                                                             bool                             blacklist,
-                                                             bool                             covered,
-                                                             const std::vector<std::string>&  metadataFilter);
+  static SenderThreadParams* buildSenderParams(ContextElementResponse*          notifyCerP,
```

Comment:
`std::vector<SenderThreadParams*>*` has been simpified to just one `SenderThreadParams*`.

Since the removal of initial notification, the vector has always one element, so now code is much simpler.

NTC (informative)